core/src/test/java/org/elasticsearch/search/aggregations/bucket/nested/NestedAggregatorTests.java

```diff
@@ -314,7 +326,7 @@ public void testResetRootDocId() throws Exception {
                 fieldType.setName(VALUE_FIELD_NAME);
 
                 BooleanQuery.Builder bq = new BooleanQuery.Builder();
-                bq.add(Queries.newNonNestedFilter(), BooleanClause.Occur.MUST);
+                bq.add(Queries.newNonNestedFilter(Version.CURRENT), BooleanClause.Occur.MUST);
```

Comment:
randomize version? The test should be able to handle both the old and new way.

Reply:
++